### PR TITLE
Update Metabase for OCP4

### PR DESCRIPTION
**Pull request by jujaga**
# Description

This PR cleans up and updates Metabase templates to be OCP4 compatible.
- Add documentation for cross-namespace NSP connectivity
- Split out secret generation into standalone template
- Update to OpenJDK 11 (same base image used by upstream Metabase)

[SHOWCASE-1860](https://apps.nrs.gov.bc.ca/int/jira/browse/SHOWCASE-1860)

## Types of changes

New feature (non-breaking change which adds functionality)
Documentation (non-breaking change with enhancements to documentation)

## Checklist

- [x] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [x] I have checked that unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Review by loneil — APPROVED** [review]
👍

File: tools/metabase/openshift/metabase.dc.yaml [hunk 1]
```diff
@@ -189,34 +188,12 @@ parameters:
...
-  - name: EMBEDDING_SECRET_KEY
-    displayName: Embedding Secret Key
-    description: A key that is used to embed metabase screens from within a website
-    generate: expression
-    from: "[a-zA-Z0-9]{16}"
+  - name: DATA_VOLUME_CAPACITY
+    description: Volume space available for Data
+    displayName: Data Volume Capacity (use M or G, not Mi or Gi)
     required: true
+    value: 1Gi
```
Comment (TimCsaky) [inline tools/metabase/openshift/metabase.dc.yaml:195]: says don't use Gi in line above (?)

**Comment by TimCsaky** [comment]
looks great. Did you try deploying this on COP4?

File: tools/metabase/openshift/metabase.dc.yaml (hunk 1 above)
Comment (jujaga) [reply tools/metabase/openshift/metabase.dc.yaml:195]: Probably ancient OCP3 specific note. It needed to be Gi for it to register. Removing that note.

**Pull request merged**